subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -30,11 +30,20 @@ LOG_MODULE_REGISTER(net_wifi_shell, LOG_LEVEL_INF);
 
 #include "net_shell_private.h"
 #ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
-#include "ca-cert.h"
-#include "client-cert.h"
-#include "client-key.h"
-#include "client-cert.h"
-#include "client-key.h"
+static const char ca_cert_test[] = {
+	#include <wifi_enterprise_test_certs/ca.der.inc>
+	'\0'
```

Comment:
@krish2718 why do we need to add `\0` here, I do not see a need for that in echo-server sample for example.

Reply:
We can remove that, I added it based on `thrift`, but it's just specific to `thrift` (https://github.com/zephyrproject-rtos/zephyr/commit/218f787e2619851ca17a67780f144c62619421ff )

Reply:
removed '\0'